mosviz/controls/slits/saved_slits.yaml

```diff
@@ -0,0 +1,60 @@
+# Last updated: 09/10/2018
+# name_of_slit:
+#   name: "Display name of slit" # Required
+#   type: MOSViz slit class
+#   length: Quantity or list # User input if not specified
+#   width: Quantity or list # User input if not specified
+
+nirspec_msa:
+  name: "NIRSpec MSA (Single Slit)"
+  type: "RectangleSkySlit"
```

Comment:
This might be better deferred to another PR if you think it'll take a lot of work... But what about having the type just be "rectangle", and using the type of units in `length` and `width` to determine if it's Sky or Pixel?  (i.e., if it's angular units it's sky, and if it's unitless assume pixel?)

Reply:
I like this, I am going to edit it now.